templates/archives/stories/mayowa_story.jinja2

```diff
@@ -26,15 +26,29 @@ Mayowa Songonuga
-    Machines. By June of 1954, automatic computation, as it was called then, had spread to many different
-    departments, including Mechanical, Chemical, and Aeronautical Engineering, Physics, and Mathematics.
-    These departments all offered classes that utilized automatic computing in some way, but there were
-    also eleven classes directly related to the emerging field. Of these eleven, ten were graduate electives,
-    and the last was a senior elective. 6.535 evolved to  Introduction to Computer Coding and Logic and
-    6.536 to Machine Computation. Other classes included M411, Numerical Analysis, and 6.568, Switching Circuits.</p>
+<p>The first high-speed computer was installed at MIT in early 1957. Before that, MIT had other, less advanced machinery
+    that was used in the classroom setting. Computer Science at MIT started as just a few classes, notably 6.535 and
+    6.536, Coding for digital Machines and Coding for Analog Machines. By June of 1954, automatic computation, as it
+    was called then, had spread to many different departments, including Mechanical, Chemical, and Aeronautical
+    Engineering, Physic, and Mathematics. These departments all offered classes that utilized automatic computing in
+    some way, but there were also eleven classes directly related to the emerging field. Of these eleven, ten were
+    graduate electives, and the last was a senior elective. 6.535 evolved to  Introduction to Computer Coding and Logic
+    and 6.536 to Machine Computation. Other classes included M411, Numerical Analysis, and 6.568, Switching Circuits. </p>
+
+<p>Currently, there are over 300 classes offered by the course 6 departments alone, not including all of the classes
+    involving computation that are offered in other departments. It’s impossible to be an MIT student without access to
+    a computer, something that couldn’t have been predicted back in the early days of the computer. Back then, things
+    were very different, even down to the academic structure of MIT. The reason MIT has courses instead of majors is
+    because every student’s course map was predecided based on what degree program they applied to. If you applied to
+    MIT for electrical engineering and were excepted, then, when you got on campus, you were told exactly what classes
+    you were going to take which semester and given very little room for elective choice. Each department literally had
+    a course planned out for their students. Now, you no longer have to declare your major before arriving on campus.
+    You have until the end of sophomore year to do it, and even within the departments, there is a lot of flexibility
+    in terms of what classes you can take to fulfill requirements and when.</p>
+
+    <p><a href="/archives/doc/1_9_cc_proposal_history_24">Click here to see the list of computer science related classes in 1954</a></p>
```

Comment:
These are great external links! Have you found anything in our archive project that you can add to this write up? Adding relevant documents from our archive showcases the value of the work we have done and adds some more insight into what is available in our archive.